task-launcher/src/tasks/shared/helpers/fetchAndParseCorpus.js

```diff
@@ -81,19 +79,39 @@ const transformCSV = (csvInput, numOfPracticeTrials, sequentialStimulus) => {
       currPracticeAmount = 0;
     }
 
-    // Only push in the specified amount of practice trials
-    if (newRow.notes !== 'practice') {
-      stimulusData.push(newRow);
-    } else if (newRow.notes === 'practice' && currPracticeAmount < numOfPracticeTrials) {
+    if (newRow.trialType === "instructions") {
+      if (task == "trog") { 
+        instructionData.push(newRow);
+      } else { 
+        stimulusData.push(newRow);
```

Comment:
this would push non-TROG 'instructions' trials into stimulusData, which is undesirable